src/mmw/js/src/core/templates/header.ejs

```diff
@@ -13,7 +13,7 @@
                     </a>
                     <ul class="dropdown-menu">
                         <li><a href="#">Profile</a></li>
-                        <li><a href="accounts/logout">Logout</a></li>
+                        <li><a href="javascript:void(0);" class="user-logout">Logout</a></li>
```

Comment:
Good point @kdeloach. There's certainly something to be said for being able to actually log out without js.